api/tests/integration/infrastructure/repositories/certification-center-repository_test.js

```diff
@@ -28,7 +28,7 @@ describe('Integration | Repository | Certification Center', () => {
         expect(certificationCenter.id).to.equal(1);
         expect(certificationCenter.name).to.equal('certificationCenterName');
         expect(certificationCenter.createdAt).to.deep.equal(new Date('2018-01-01T05:43:10Z'));
-        expect(certificationCenter).to.have.all.keys(['id', 'name', 'externalId', 'createdAt']);
+        expect(certificationCenter).to.have.all.keys(['id', 'name', 'type', 'externalId', 'createdAt']);
```

Comment:
expect(certificationCenter).to.be.an.instanceof(CertificationCenter) ?

Reply:
up @dherault :)